### PR TITLE
TMDM-13091 Remove the menu of system role can still access it from welcome page

### PR DESCRIPTION
**Pull request by yjli-talend**
https://jira.talendforge.org/browse/TMDM-13091
**What is the current behavior?** (You should also link to an open issue here)

**What is the new behavior?**
1 `Getting Started` portlet will display `Master Data Browser` if menu has `Master Data Browser` item.
2 `Getting Started` portlet will display `Journal` if menu has `Journal` item.
3 `Search` portlet will display if menu has `Search` item.

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: org.talend.mdm.webapp.general/src/main/java/org/talend/mdm/webapp/general/client/layout/PortletConfigFieldSet.java [hunk 1]
```diff
@@ -78,6 +82,8 @@
 
     private int colNum = 3;
 
+    private List<String> menus = new ArrayList<String>();
```
Comment (yjli-talend) [inline org.talend.mdm.webapp.general/src/main/java/org/talend/mdm/webapp/general/client/layout/PortletConfigFieldSet.java]: Gwt sdk can't support grammar like `List<String> menus = new ArrayList<>()`

File: org.talend.mdm.webapp.general/src/main/java/org/talend/mdm/webapp/general/client/layout/PortletConfigFieldSet.java [hunk 2]
```diff
@@ -297,8 +315,10 @@ private void updatePortletConfig(String dataString) {
             if (alertWidgetCheckBox.getValue()) {
                 updates.add(PortletConstants.ALERT_NAME);
             }
-            if (searchWidgetCheckBox.getValue()) {
-                updates.add(PortletConstants.SEARCH_NAME);
+            if (menus.contains(SEARCH_CONTEXT)) {
+                if (searchWidgetCheckBox.getValue()) {
```
Comment (VictorLi-talendbj) [inline org.talend.mdm.webapp.general/src/main/java/org/talend/mdm/webapp/general/client/layout/PortletConfigFieldSet.java]: How about using one `if(... && ...)` condition?

File: org.talend.mdm.webapp.general/src/main/java/org/talend/mdm/webapp/general/server/actions/GeneralAction.java [hunk 3]
```diff
@@ -107,6 +106,18 @@ public MenuGroup getMenus(String language) throws ServiceException {
...
     }
 
+    @Override
+    public List<MenuBean> getMenuList(String language) throws ServiceException {
+        try {
+            List<MenuBean> menus = new ArrayList<>();
+            Utils.getSubMenus(Menu.getRootMenu(), language, menus, 1, 1);
+            return menus;
+        } catch (Exception e) {
+            LOG.error(e.getMessage(), e);
```
Comment (lpw82gf) [inline org.talend.mdm.webapp.general/src/main/java/org/talend/mdm/webapp/general/server/actions/GeneralAction.java]: Provide some meaningful self-defined message here, don't just use `e.getMessage()`, do it for all exception log.

File: org.talend.mdm.webapp.welcomeportal/src/main/java/org/talend/mdm/webapp/welcomeportal/client/MainFramePanel.java [hunk 4]
```diff
@@ -268,6 +280,15 @@ protected void doOnFailure(Throwable caught) {
         }
     }
 
+    private void initProltes() {
```
Comment (VictorLi-talendbj) [inline org.talend.mdm.webapp.welcomeportal/src/main/java/org/talend/mdm/webapp/welcomeportal/client/MainFramePanel.java]: `initProltes' -> `initPortlets`?

**Pull request merged**